apps/dsp-app/cypress/support/helpers/constants.ts

```diff
@@ -0,0 +1,36 @@
+export enum ClassType {

```

Comment:
why are there enums in a constants.ts file ? (should be only constants)

Reply:
I prefer to have enums and constants in the same file, it will be used just for e2e. So far there is no constants yet.